core/templates/pages/exploration-editor-page/statistics-tab/templates/playthrough-modal.template.html

```diff
@@ -28,3 +28,20 @@ <h3 class="modal-title">Sample Playthrough</h3>
     Return
   </button>
 </div>
+<style>
+  .oppia-improvement-playthrough-modal .modal-body {
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+    padding-bottom: 8rem;
+  }
+  .oppia-improvement-playthrough-modal .oppia-playthrough-modal-arrow-btn {
+    display: block;
```

Comment:
Can we use the boostrap class d-block?
https://getbootstrap.com/docs/4.0/utilities/display/

Reply:
done.